src/main/java/com/softwaredesign/getHostsIpAndMac.java

```diff
@@ -0,0 +1,74 @@
+package com.softwaredesign;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ConnectException;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.SocketTimeoutException;
+import java.util.ArrayList;
+import java.util.List;
+//gets the Ip Address of the host computer and the MAC address of that computer 
+//protocol for mapping an Ip address to a physical MAC address on a local network
+//Every computer is assigned an IP address and some local server's IP adresses a
+public class getHostsIpAndMac { 
```

Comment:
Naming convention states class name to start with a capital. Something like "HostIPRetriever" or MACRetriever / Fetcher